src/pages/Withdrawal/index.tsx

```diff
@@ -179,29 +168,35 @@ const Withdrawal: React.FC<WithdrawalProps> = ({
       } catch (_) {
         throw IncorrectPINError;
       }
+      // Check if substract fee necessary
+      const LbtcBalance = balances.find((b) => b.coinGeckoID === 'bitcoin')?.amount || 0;
+      const fee = estimateFeeAmount(utxos, [getRecipient()], 0.1, () => null);
+      const needLBTC = toSatoshi(LbtcBalance.toString(), 8, lbtcUnit).plus(fee).toNumber();
+      const substractFee = needLBTC > LbtcBalance;
```

Comment:
This check is not necessary. My bad if the argument name in sendTx is not enough clear. You must read it as "subtract fee **if and only if** the user inputs an amount greater than available balance". It means that this check is done inside the `sendTx` function. In other words, the `substractFee` var [should be a constant set to true.](https://github.com/vulpemventures/marina/pull/244/commits/9680c4f0e4ebefc534069163f78afb636f26008c#diff-a9c34e278ae2836b9ea141dd8456cc9157aed7fd7a0e0d0b18fcc7fb1ea2c16cR284)

Reply:
Ha ok ok

Reply:
done